packages/stylelint-config/stylelint.config.js

```diff
@@ -12,5 +12,6 @@ module.exports = {
     "color-format/format": { format: "rgb" },
     "plugin/declaration-block-no-ignored-properties": true,
     "order/properties-order": require("./properties-order"),
+    "alpha-value-notation": "number",
```

Comment:
Also need to update the peer dependency to 14.2.0 under the `packages/stylelint-config/package.json` so this works on ~jobber online as well~ any project that uses `@jobber/stylelint`. _Which, I just found out that isn't really used in Jobber Online_ 😨 

https://github.com/GetJobber/atlantis/blob/1146169ab41ee01106d292f8722ef498f581d629/packages/stylelint-config/package.json#L16-L18